### DISCUSSION TITLE
TP5 - typedef struct

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#157] [Preguntas y respuestas] [answered]
¿Puedo definir `typedef struct fraccion` fuera del `main`?

**Comment by martinvilu** [comment, marked as answer]
Va fuera del `main`,

```C
#include <stdio.h>

typedef struct cosa
{
    int cosita;
    double cosota;
} cosa_t;

int main()
{
    cosa_t cosa_variable;
}
```

**Reply by rinrot** [reply]
gracias por clarificar profe

**Reply by martinvilu** [reply]
![st,small,507x507-pad,600x600,f8f8f8](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/34cf5ab0-c4bb-4447-aa50-106310b906c6)
